doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -246,148 +173,57 @@ nRF IEEE 802.15.4 radio driver
 Thread
 ------
 
-* Added:
-
-  * Support for storing the Thread key materials in the :ref:`ug_ot_thread_security_kmu`.
-  * The :ref:`ug_ot_thread_security` user guide describing the security features of the |NCS| OpenThread implementation.
+|no_changes_yet_note|
 
 Zigbee
 ------
 
-* Removed all Zigbee resources.
-  They are now available as separate `Zigbee R22`_ and `Zigbee R23`_ add-on repositories.
+|no_changes_yet_note|
 
 Wi-Fi®
 ------
 
-* The :ref:`ug_wifi_regulatory_certification` documentation is now moved under :ref:`ug_wifi` protocol page.
+|no_changes_yet_note|
 
 Applications
 ============
 
-* Removed the Asset Tracker v2 application.
-  For development of asset tracking applications, refer to the `Asset Tracker Template <Asset Tracker Template_>`_.
-
-  The factory-programmed Asset Tracker v2 firmware is still available to program the nRF91 Series devices using the `Programmer app`_, `Quick Start app`_ , and the `Cellular Monitor app`_.
+|no_changes_yet_note|
 
 Machine learning
 ----------------
 
-* Updated the application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board.
+|no_changes_yet_note|
 
 Asset Tracker v2
```

Comment:
I guess we could get rid of this section as well, as the application is now removed.